samples/performance/command_buffer_usage/command_buffer_usage_tutorial.md

```diff
@@ -40,45 +93,44 @@ This flag however will force separate internal allocators to be used for each co
 This is the worst-performing method of managing command buffers as it involves a significant CPU overhead for allocating and freeing memory frequently.
 The sample shows how to use the framework to follow this approach and profile its performance.
 
-![Allocate and Free](images/allocate_and_free.jpg)
+![Allocate and Free](images/allocate_and_free.png)
 
 Rather than freeing and re-allocating the memory used by a command buffer, it is more efficient to recycle it for recording new commands.
 There are two ways of resetting a command buffer: individually, with [vkResetCommandBuffer](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkResetCommandBuffer.html), or indirectly by resetting the command pool with [vkResetCommandPool](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkResetCommandPool.html).
 
-## Resetting individual command buffers
+### Resetting individual command buffers
 
 In order to reset command buffers individually with [vkResetCommandBuffer](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkResetCommandBuffer.html), the pool must have been created with the [RESET_COMMAND_BUFFER_BIT](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkCommandPoolCreateFlagBits.html) flag set.
 The buffer will then return to a recordable state and the command pool can reuse the memory it allocated for it.
 
 However frequent calls to [vkResetCommandBuffer](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkResetCommandBuffer.html) are more expensive than a command pool reset.
 
-![Reset Buffers](images/reset_buffers.jpg)
+![Reset Buffers](images/reset_buffers.png)
```

Comment:
3 images on this page are still .jpg and aren't displaying as it

Reply:
thanks!